### ISSUE TITLE
Burst documentation

### ISSUE THREAD

**Issue by brprkr** [antifuchs/governor#184]
Thanks for this amazing crate!

Struct `governor::Quota`:
> "(..) However, the quota of Quota::per_minute(60) has a burst size of 60 cells, meaning it is possible to accommodate 60 cells in one go, followed by a minute of waiting."

Based on the wording, it may be misunderstood that after the initial burst, the limiter will enforce a one-minute wait, during which time no cells will be allowed through. Would you be willing to consider rephrasing it as: "... meaning it is possible to accomodate 60 cells in one go, after which the equivalent of a minute of inactivity is required for the burst allowance to be fully restored?"

The sentence appearing later makes the intention clear: "In other words, the burst size is the maximum number of cells that the rate limiter will ever allow through without replenishing them." In my opinion it's just hard to parse if you've already misread the above. The misconception may be more likely for users like me approaching from the angle of average/max events per period rather than cell replenishment.

**Comment by antifuchs** [comment]
You're right, thanks for reading this so closely & for the suggestion! I'll make the docs improvement.

**Comment by brprkr** [comment]
Thank you!
